package.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "hawk.admin",
+    "description": "Hawk admin",
+    "version": "0.0.1",
+    "repository": "https://github.com/codex-team/hawk.admin.git",
+    "main": "src/app.js",
+    "license": "MIT",
+    "scripts": {
+        "dev": "./node_modules/.bin/nodemon ./src/app.js"
```

в разделе скриптов можно не указывать полный путь
```suggestion
        "dev": "nodemon ./src/app.js"
```